### ISSUE TITLE
Hide lower third H2R not working[BUG]

### ISSUE THREAD

**Issue by Dmissen** [bitfocus/companion-module-h2r-graphics#8]
Hi, anyone else having issues with the hide lower third function for H2R? all have features with companion and H2R work fine

**Comment by arne182** [comment]
Yes, Same problem.

**Comment by josephdadams** [comment]
What version of Companion are you running? Because this was fixed in a commit back in December.

**Comment by iamjohnbarker** [comment]
Yes, this was a known issue that has since been fixed, however I am not sure if that fix made it into an official build yet.

**Comment by arne182** [comment]
Tested not working on 2.1.2-fbcaf3-2679

**Comment by arne182** [comment]
Can confirm on v2.2.0-49b4127 it is working

**Comment by arne182** [comment]
![image](https://user-images.githubusercontent.com/844102/107115176-13541900-686b-11eb-95b0-8314b1af3840.png)
20 Dec 2020 the fix was updated but the release of 2.1.2 was on the 6 Dec 2020
![image](https://user-images.githubusercontent.com/844102/107115180-1c44ea80-686b-11eb-9226-2e70094a5ff7.png)
So it is not yet in the release.
